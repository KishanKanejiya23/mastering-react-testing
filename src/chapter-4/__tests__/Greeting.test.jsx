import React from 'react'
import renderer from 'react-test-renderer'
import Greeting from '../Greeting'

test('Greeting component renders correctly', () => {
  const tree = renderer.create(<Greeting name="John Doe" />).toJSON()
  expect(tree).toMatchSnapshot()
})
