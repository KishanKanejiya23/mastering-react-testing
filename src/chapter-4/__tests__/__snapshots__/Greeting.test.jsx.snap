// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting component renders correctly 1`] = `
<h1>
  Hello, 
  John Doe
  !
</h1>
`;
